spec/controllers/projects_controller_spec.rb

```diff
@@ -193,6 +193,19 @@
               expect(assigns[:story].project).to eq(project)
             end
           end
+
+          describe 'when the user change to another project from another team' do
+            let(:new_team)            { create :team }
+            let(:new_project)         { create :project, users: [user] }
+            let(:new_story)           { create :story, project: new_project, requested_by: user }
+            it 'should accept request when it is from registred team' do
```

Comment:
```suggestion
            it 'should accept request when it is from registred team', :aggregate_failures do
```

Also, remember to use `aggregate_failures` when using more than one `expect`.

Reply:
I didn't know that :thinking: